bin/check_exercises.sh

```diff
@@ -1,66 +1,45 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# can't benchmark with a stable compiler; to bench, use
-# $ BENCHMARK=1 rustup run nightly bin/check_exercises.sh
-if [ -n "$BENCHMARK" ]; then
-    target_dir=benches
-else
-    target_dir=tests
-fi
-
 repo=$(git rev-parse --show-toplevel)
 
 if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
     git fetch --depth=1 origin main
     files="$(
         git diff --diff-filter=d --name-only remotes/origin/main |
-        grep "exercises/" || true |
-        cut -d '/' -f -3 |
-        sort -u |
-        awk -v repo="$repo" '{print repo"/"$1}'
+        grep "exercises/" |
+        cut --delimiter '/' --fields -3 |
+        sort --unique || true
     )"
 else
     files="$repo/exercises/*/*"
 fi
 
-return_code=0
 # An exercise worth testing is defined here as any top level directory with
 # a 'tests' directory and a .meta/config.json.
 for exercise in $files; do
-    exercise="$exercise/$target_dir"
-
-    # This assumes that exercises are only one directory deep
-    # and that the primary module is named the same as the directory
-    directory=$(dirname "${exercise}")
+    slug=$(basename "$exercise")
```

Comment:
I made it so the `./bin/test_exercise.sh` script accepts a slug instead of a path. Makes it a little more convenient to test specific exercises one is working on.